### ISSUE TITLE
Don't break testing of tarballs

### ISSUE THREAD

**Issue by Cogitri** [rhysd/cargo-husky#2]
Running `cargo test` from tarballs (e.g. the ones from crates.io) is broken when using cargo-husky for that project:

```
error: failed to run custom build command for `cargo-husky v1.2.0`
process didn't exit successfully: `/builddir/tmplgen-0.4.0/target/release/build/cargo-husky-19c04c7d2358a007/build-script-build` (exit code: 1)
--- stderr
Error: .git directory was not found in '/builddir/tmplgen-0.4.0/target/release/build/cargo-husky-9771d89858eeb562/out' or its parent directories

warning: build failed, waiting for other jobs to finish...
error: build failed
```

I suppose it would be best to just do nothing if cargo-husky is run in a folder which isn't a git repo.

**Comment by rhysd** [comment]
I did not care about the case. Thanks for your report.
Could you tell me the steps to reproduce this to avoid misunderstanding? I'm not understanding steps after downloading tarball of some package from crates.io.

**Comment by Cogitri** [comment]
1. Download a crate which uses cargo-husky 
e.g. https://static.crates.io/crates/tmplgen/tmplgen-0.4.0.crate

2. Unpack the crate

3. Run `cargo test` for the crate. This fails because the tarball doesn't contain the `.git` repository

**Comment by rhysd** [comment]
Thank you. I'll try it tonight

**Comment by rhysd** [comment]
This should be fixed at ce08d9dbb18c9f447113fe28056f961eecf716a0. Thank you again for your report.
